scripts/teamDataScript.js

```diff
@@ -345,7 +345,10 @@ function loadMatch()
 
     var customNotesInputHTML = `<textarea class="match-textAreas match-frequency js_clear_on_load"></textarea>`; //for custom notes each match
     var customNotesRow = `<tr> <td class="match-element scoutingTableRow">Custom Notes</td> <td class="scoutingTableRow">${customNotesInputHTML}</td> </tr>`;
+    var RPInputHTML = `<input style = "width: 80%" type="number" class="match-frequency js_clear_on_load">`; //for ranking points each match
+    var RPRow = `<tr> <td class="match-element scoutingTableRow">Ranking Points</td> <td class="scoutingTableRow">${RPInputHTML}</td> </tr>`;
```

Comment:
Not everyone is going to want to "track" ranking points (they can see who's ranked highest through lots of other means) - we should probably just tell users who want this to utilize the "Numeric" type input.